recipes/spacy-loggers/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "spacy-loggers" %}
+{% set modulename = "spacy_loggers" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8415bda3d376fc2ce23910c28fb86fbd6ab3349804aaa525f78e8b533b69a299
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - {{ modulename }}
+  requires:
+    - pytest
+    - spacy >=3.1
+  commands:
+    - python -m pytest --tb=native --pyargs {{ modulename }}
+
+
+about:
+  home: https://github.com/explosion/spacy-loggers
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Alternate loggers for spaCy pipeline training
+  description: |
+      Enables alternatives to spaCy's built-in console logger,
+      particularly the Weights and Biases logger.
```

Comment:
This will get stale. I'd suggest removing the description?

```suggestion
```

Reply:
Maybe just "Enables alternatives to spaCy's built-in console logger", then?